abci/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "tendermint-abci"
```

```suggestion
name = "abci"
```

it can be called just this and published as a new breaking version of https://crates.io/crates/abci -- @xla @liamsi can help with that